defaults/main.yml

```diff
@@ -1,9 +1,11 @@
 ---
 ## Default Variables for ansible-dkim role
 
-### Opendkim debian/ubuntu package configuration
+### Opendkim package configuration
+# - Redhat : /etc/opendkim.conf
+# - Debian / default configuration default to : /etc/default/opendkim
+dkim_default_config_file: "{{ '/etc/opendkim.conf' if ansible_distribution_file_variety == 'RedHat' else '/etc/default/opendkim' }}"
```

Comment:
I believe that `/etc/opendkim.conf` is main config file, not `defaults` as they are in Debian/Ubuntu. It would be probably better idea make distro conditions to commands working with defaults file?

Reply:
Also, is `ansible_distribution_file_variety` proper var to test? Why you choosed this one?

Reply:
Thanks for the feedback!  
I'll make /etc/opendkim.conf as default. 

After further testing, I think using `ansible_os_family` would be better suited. 
`ansible_distribution_file_variety` would produce Centos/Alma/Rocky on Redhat derivatives. 
 
Mapping for OS family : https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/facts/system/distribution.py#L511

I don't have much exposure to Debian derivatives so can't comment on that.